lib/models/noSignal.js

```diff
@@ -41,6 +41,7 @@ var util = require('util'),
     ID_REGEXP = 5,
```

Comment:
Is this PR the fix for issue #253? It should be mentioned in the CNR file (with a reference to issue number)